src/gcodeExport.cpp

```diff
@@ -602,10 +630,10 @@ void GCodeExport::writeTravel(int x, int y, int z, double speed)
     CommandSocket::sendLineTo(travel_move_type, Point(x, y), display_width);
 
     *output_stream << "G0";
-    writeFXYZE(speed, x, y, z, current_e_value);
+    writeFXYZE(speed, x, y, z, current_e_value, PrintFeatureType::MoveCombing);
```

Comment:
I'd rather see travel time as a global thing; It's good to know that for a given print, Cura spent X time doing travels.